nservicebus/azure-service-bus/performance-tuning_body_asb_[,7).partial.md

```diff
@@ -0,0 +1,14 @@
+### CPU vs IO bound processing
+
```

Comment:
TBH you can push it, to around 400/s. But you need to do the sends in parallel yourself